auto_nag/user_activity.py

```diff
@@ -4,90 +4,97 @@
 
 from enum import Enum
 
-from libmozdata.bugzilla import Bugzilla, BugzillaUser
+from libmozdata import utils as lmdutils
+from libmozdata.bugzilla import BugzillaUser
 
 from auto_nag import utils
+from auto_nag.people import People
 
 DEFAULT_ACTIVITY_WEEKS = 26
+DEFAULT_ABSENT_WEEKS = 13
 
 
 class UserStatus(Enum):
     UNDEFINED = 1
     DISABLED = 2
     INACTIVE = 3
+    ABSENT = 4
 
 
 class UserActivity:
-    def __init__(self, weeks_count=DEFAULT_ACTIVITY_WEEKS) -> None:
-        self.weeks_count = weeks_count
+    def __init__(
+        self,
+        activity_weeks_count=DEFAULT_ACTIVITY_WEEKS,
+        absent_weeks_count=DEFAULT_ABSENT_WEEKS,
+    ) -> None:
+        self.activity_weeks_count = activity_weeks_count
+        self.absent_weeks_count = absent_weeks_count
+        self.people = People.get_instance()
+
+        self.activity_limit = lmdutils.get_date("today", self.activity_weeks_count * 7)
+        self.seen_limit = lmdutils.get_date("today", self.absent_weeks_count * 7)
 
     def check_users(self, user_emails):
-        user_emails = set(user_emails)
+        user_emails = self.get_not_employees(user_emails)
 
-        none_users = {
-            user_email for user_email in user_emails if utils.is_no_assignee(user_email)
+        status = {
```

Comment:
Nit: `status` is a bit confusing, could you call it something like `user_to_status` or `user_statuses` or `status_map` or ...?